js/test/daterange/testdatefmtrange_fmttemplate.js

```diff
@@ -2795,7 +2795,7 @@ module.exports.testdaterangefmt = {
         test.equal(templatec11[0], '{sm} {sd} – {em} {ed}, {ey}');
         test.equal(templatec11[1], '{sm} {sd} – {em} {ed}, {ey}');
         test.equal(templatec11[2], '{sm} {sd} – {em} {ed}, {ey}');
-        test.equal(templatec11[3], '{sm}/{sd}/{sy} – {em}/{ed}/{ey}');
+        test.equal(templatec11[3], '{sm}/{sd} – {em}/{ed}/{ey}');
```

Comment:
According to spec documentation, https://github.com/iLib-js/iLib/blob/development/docs/DateRangeFormatSpec.md
cFmt11, short length daterange format is
`c11 | short | Mdy –Mdy`
Do you want to change this part?

Reply:
In the {date}{time} version of Mdy (line 56), it used `Mdy - Mdy` and in the {time}{date} version of Mdy (line 119), it used `Md - Mdy`. One or the other was wrong! I thought the one on line 56 was the one that was wrong because the year is supposed to be the same for the c11 format, so it makes sense to share it. I will update this spec to match.